apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -133,9 +133,32 @@ export default function Rundown(props: RundownProps) {
             break;
           }
         }
+      } else if (event.altKey && event.ctrlKey && !event.shiftKey) {
+        switch (event.code) {
+          case 'ArrowDown': {
+            if (entries.length < 2 || cursor == null) {
```

Comment:
how come the check for `ArrowUp` and `ArrowDown` is the same?

Reply:
was just using the same switch case approach as the one above if more shortcuts were added
but I can collapse it for now 👍🏽

Reply:
ah ok, you dont have to change it. if the logic is the same then thats it

I would have just expected that the logic would have been different, so when merging up we need to check if there is something up, when merging down we need to check if there is something down